### ISSUE TITLE
Is there a plan to add support for node.js?

### ISSUE THREAD

**Issue by jainanshul** [fluffynukeit/jMatFile#1]

**Comment by fluffynukeit** [comment]
No.  I don't remember the details, but I believe there was something about the supported API not being available in nodejs, at least at the time I wrote it.  I don't use this repo at all these days, so I don't expect any active development on my part, but I will merge reasonable PRs.
